protos/guard/guard.proto

```diff
@@ -119,18 +128,22 @@ message FileStoreMeta {
 }
 
 message FileStoreStatus {
-  FileStoreMeta file_store_meta = 1;
+  FileStoreMeta file_store_meta = 1 [
+    (gogoproto.embed) = true,
+    (gogoproto.nullable) = false
+  ];
   enum MetaState {
     DRAFT = 0; //the proposal was submit by renter
     UPLOADING = 1; //all related contracts were signed, and renter is uploading shards and questions
     RUNNING = 2; //guard complete the first challenge for all shards
     PARTIAL_LOST = 3; //some shards were lost, but the whole file is safe
     LOST = 4; //some shards were lost and the whole file cannot be recovered now
     DONE = 5; //all contracts were done
+    CANCELED = 6; //reserved for the cancel contract
   }
   MetaState state = 2;
-  repeated Contract contracts = 3;
-  bytes renter_signature = 4; //renter's signature for FileStoreMeta and all contractMeta
+  repeated Contract contracts = 3 [(gogoproto.nullable) = false];
```

Comment:
This should not use nullable. Weiyu's comment only applies to embedded struct pointer.

Reply:
This generates `[]*Contract` which is correct

Reply:
I see, struggled for it now, and will do the change